Sources/Auth/AuthClientFactory.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import WalletConnectRelay
+import WalletConnectUtils
+import WalletConnectKMS
+import WalletConnectPairing
+
+public struct AuthClientFactory {
+
+    public static func create(metadata: AppMetadata, account: Account?, relayClient: RelayClient) -> AuthClient {
+        let logger = ConsoleLogger(loggingLevel: .off)
+        let keyValueStorage = UserDefaults.standard
+        let keychainStorage = KeychainStorage(serviceIdentifier: "com.walletconnect.sdk")
+        return AuthClientFactory.create(metadata: metadata, account: account, logger: logger, keyValueStorage: keyValueStorage, keychainStorage: keychainStorage, relayClient: relayClient)
+    }
+
+    static func create(metadata: AppMetadata, account: Account?, logger: ConsoleLogging, keyValueStorage: KeyValueStorage, keychainStorage: KeychainStorageProtocol, relayClient: RelayClient) -> AuthClient {
```

Comment:
why account is optional? Is it possible to avoid it?

Reply:
yes, App do not need account